dev/com.ibm.ws.rest.handler.validator.cloudant_fat/bnd.bnd

```diff
@@ -27,6 +27,7 @@ fat.test.container.images: kyleaure/cloudant-developer:1.0
 -buildpath: \
 	com.ibm.websphere.javaee.servlet.3.1;version=latest,\
 	com.ibm.websphere.javaee.jsonp.1.0;version=latest,\
-	io.openliberty.org.testcontainers;version=latest
+	io.openliberty.org.testcontainers;version=latest,\
+	io.openliberty.com.google.gson;version=latest
```

Comment:
question: what is the additional dependency on gson used for? I don't see any source code updates that require gson.

Reply:
Not a new dependency; an existing dependency that does not resolve properly for multi-threaded builds. I believe that is disabled except for local builds, and is often disabled for MacOS.  With a multi-thread build of just this component, it seems that most of the time the `io.openliberty.com.google.gson` project is not built in the correct order, so the build fails attempting to find that project.  By adding the project to the build path, this coerces the build to ensure the `io.openliberty.com.google.gson` is always processed prior to the build.gradle step that needs it.

In build.gradle, the following cloudant depenencies are defined:
```
cloudantCommonDeps  project(':com.ibm.ws.org.apache.commons.io'),
      					  project(':io.openliberty.com.google.gson')
```
And this appears to be fine for some builds, I suspect whenever multi-threaded building is disabled; however, when building just the `com.ibm.ws.rest.handler.validator.cloudant_fat` project by itself on a windows system with multi-threaded builds enabled, the build is unable to find the `io.openliberty.com.google.gson` project.